src/libs/sfdk/virtualmachine.h

```diff
@@ -126,6 +126,10 @@ class SFDK_EXPORT VirtualMachine : public QObject
     void setStorageSizeMb(int storageSizeMb, const QObject *context,
             const Functor<bool> &functor);
 
+    void currentFreeSize(const QObject *context, const Functor<int> &functor);
+    void setFreeSizeMb(int freeSizeMb, int incrementMb, bool allowShrink,
+            const QObject *context, const Functor<bool> &functor);
+
```

Comment:
It makes me ask how does `incrementMb` affect shrinking? So I would rather keep this API grow-only and let sfdk fail if '+' is not used right now (shrinking not supported by any of our VMs yet).

I would prefer `storageFreeSizeMb` and `ensureStorageFreeSizeMb` as the names and stick them together with the above functions as in:

```
storageSizeMb
freeStorageSizeMb
setStorageSizeMb
ensureStorageSizeMb
```

Reply:
> It makes me ask how does `incrementMb` affect shrinking? So I would rather keep this API grow-only and let sfdk fail if '+' is not used right now (shrinking not supported by any of our VMs yet).

incrementMb disables shrinking. Or at least that was my plan. When I realized shrinking isn't supported I stopped worrying about it. Sure, I can remove shrinking from the API, it certainly makes things simpler.